src/constants.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable max-len */
```

Comment:
Avoid using 'eslint-disable' for specific rules. Consider adjusting the max-len rule in your ESLint configuration to an appropriate value.